client.go

```diff
@@ -214,6 +214,25 @@ func (client *Client) RangeWithOptions(key string, fromTimestamp int64, toTimest
 	return
 }
 
+// ReverseRangeWithOptions - Query a timestamp range on a specific time-series in reverse order
+// args:
+// key - time-series key name
+// fromTimestamp - start of range. You can use TimeRangeMinimum to express the minimum possible timestamp.
+// toTimestamp - end of range. You can use TimeRangeFull or TimeRangeMaximum to express the maximum possible timestamp.
+// rangeOptions - RangeOptions options. You can use the default DefaultRangeOptions
+func (client *Client) ReverseRangeWithOptions(key string, fromTimestamp int64, toTimestamp int64, rangeOptions RangeOptions) (dataPoints []DataPoint, err error) {
```

Comment:
@filipecosta90 don't you feel like ReverseRangeWithOptions and RangeWithOptions have duplicate code? can't you create a generic method that gets "TS.REVRANGE"/"TS.RANGE" and make these two call it?

Reply:
@gkorland I my opinion, given that they are two distinct commands I would not merge the method on the client. I would avoid logic on the client code that either issues command A or B depending on a flag. WDYT?

Reply:
@gkorland now that I've checked your PR I fully get the point on it ( I was thinking about exposing the same method that would issue different commands ). Totally agree on internal code reuse :)